### PR TITLE
NOISSUE - Switch to Docusaurus

### PR DESCRIPTION
**Pull request by SammyOina**
# What type of PR is this?
this a refactor of docs

## What does this do?
switches from mkdocs to docasaurus

### CODE REVIEW

**Comment by dborovcanin** [comment]
@SammyOina Is this ready?

File: README copy.md
Comment (WashingtonKK) [inline README copy.md:1]: Is this a duplicate file?

**Review by dborovcanin — CHANGES_REQUESTED** [review]
@SammyOina Let's remove the landing page and go directly to /intro.

**Comment by dborovcanin** [comment]
@SammyOina Please completely remove blogs from the navigation.

File: README.md [hunk 1]
```diff
@@ -4,50 +4,30 @@ This repo collects the collaborative work on Cocos AI documentation. Official do
...
-   source venv/bin/activate
-   ```
-
-3. Install [MkDocs](https://www.mkdocs.org/#installation)
-
-   ```bash
-   pip install mkdocs
-   ```
+```
+$ yarn
```
Comment (WashingtonKK) [inline README.md:16]: Add npm option

File: README.md [hunk 2]
```diff
@@ -4,50 +4,30 @@ This repo collects the collaborative work on Cocos AI documentation. Official do
...
+```
 
-4. Additionally, install [Material theme](https://squidfunk.github.io/mkdocs-material/):
+### Local Development
 
-   ```bash
-   pip install mkdocs-material
-   ```
+```
+$ yarn start
```
Comment (WashingtonKK) [inline README.md:22]: This does not work

**Review by WashingtonKK — CHANGES_REQUESTED** [review]
Also update instructions to run locally.

File: README.md (hunk 1 above)
Comment (SammyOina) [reply README.md:16]: we do not need both

File: README.md (hunk 2 above)
Comment (SammyOina) [reply README.md:22]: check on your local deployment

**Review by WashingtonKK — APPROVED** [review]

**Review by WashingtonKK — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**